packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/tests/rich-text-mention-users-view-matrix.stories.ts

```diff
@@ -0,0 +1,69 @@
+import { ViewTemplate, html } from '@microsoft/fast-element';
+import type { Meta, StoryFn } from '@storybook/html';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../../../../utilities/tests/matrix';
+import { createMatrixThemeStory } from '../../../../../utilities/tests/storybook';
+import { richTextMentionUsersViewTag } from '..';
+import {
+    disabledStates,
+    type DisabledState
+} from '../../../../../utilities/tests/states';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderColor,
+    borderWidth,
+    mediumPadding,
+    smallPadding
+} from '../../../../../theme-provider/design-tokens';
+
+const metadata: Meta = {
+    title: 'Tests/Rich Text Mention: User',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+
+export default metadata;
+
+const component = ([
+    disabledName,
+    disabled
+]: DisabledState): ViewTemplate => html`
+    <style class='code-hide'>
+        .mention-container {
+            font: var(${bodyFont.cssCustomProperty});
+            color: var(${bodyFontColor.cssCustomProperty});
+            margin: var(${smallPadding.cssCustomProperty});
+        }
+    </style>
+    <div class="mention-container">
```

Comment:
When first looking at the matrix test in storybook, the `[Mention View]` and `[View Disabled]` text looked odd to me. Can you comment on its purpose and whether there's a reason to use the [square bracket] formatting right next to the control?

I agree it is valuable to include text that describes the purpose of each test. But I would vote to format it nicely and either make it plain text with no square brackets (like in the spinner matrix test) or put it in the component itself (many of our tests use the label or content; in this case you could set the description in `mention-label`)

Reply:
Goal is to have some text around the inline text so you can play with selection, etc. Similar to the [anchor story](https://60e89457a987cf003efc0a5b-higxxqmccn.chromatic.com/?path=/docs/components-anchor--docs)

Removing the brackets would be fine, just should have something around and not just inside

Reply:
I removed the brackets and formatted them as meaningful texts.